src/main/java/duke/command/ToDoCommand.java

```diff
@@ -0,0 +1,37 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.task.Todo;
+import duke.ui.Ui;
+
+/**
+ * Contains the executables when the user uses the 'Todo' command.
+ */
+public class ToDoCommand extends Command {
+
+    private String task;
+
+    /**
+     * Constructor for the to do command
+     * @param task to do task to be added to the list.
+     */
+    public ToDoCommand(String task) {
+        this.task = task;
+    }
+
+    /**
+     * Executes the to do command.
+     */
+    @Override
+    public String execute(TaskList taskList, Ui ui, Storage storage) {
```

Comment:
perhaps having a more specific name like "executeTask" could be more useful for other coders to understand; honestly, i'm nitpicking at this point.